test/spec/modules/quantcastBidAdapter_spec.js

```diff
@@ -221,7 +225,11 @@ describe('Quantcast adapter', function () {
 
     it('overrides video parameters with parameters from adunit', function() {
       setupVideoBidRequest({
```

Comment:
this test looks good as is, it has two different values for mimes and you're testing the ad unit wins.